lib/manager/components/validation/MobilityDataValidationResult.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable no-fallthrough */
+import React, { useState } from 'react'
+import Icon from '@conveyal/woonerf/components/icon'
+import { ListGroupItem, Table } from 'react-bootstrap'
+import Markdown from 'markdown-to-jsx'
+
+import toSentenceCase from '../../../common/util/to-sentence-case'
+import {
+  mobilityDataValidationErrorMapping,
+  validationErrorIconLookup
+} from '../../util/version'
+
+import rules from './rules.json'
+
+// from https://stackoverflow.com/a/4149671
+function unCamelCase (s) {
+  return s
+    .split(/(?=[A-Z])/)
+    .join(' ')
+    .toLowerCase()
+}
+
+const NoticeTable = ({ headerOverides = {'stopSequence1': 'Stop seq-uence 1', 'stopSequence2': 'Stop seq-uence 2'}, notices }) => {
```

Comment:
Can we add an override for the expiry warnings? They have the number unspaced in the header: 
![image](https://user-images.githubusercontent.com/63798641/226714779-a393229a-4b37-4b1d-aea4-fb21764a97cb.png)
Which ends up looking like this: 
<img width="436" alt="image" src="https://user-images.githubusercontent.com/63798641/226714871-be5db291-a0c2-498b-911c-3f8853556094.png">